### PR TITLE
STP_EDGE

### PR DESCRIPTION
**Pull request by radez**
Need a way to enable arbitrary config options passed to the ansible roles. Start with immediate STP forwarding. This is called STP edge in some platforms

### CODE REVIEW

File: network_runner/api.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def delete_vlan(self, hostname, vlan_id):
         """
         return self.play('delete_vlan', hostname, {'vlan_id': vlan_id})
 
-    def conf_access_port(self, hostname, port, vlan_id):
+    def conf_access_port(self, hostname, port, vlan_id, portfast=False):
```
Comment (michaeltchapman) [inline network_runner/api.py]: I think we should consider whether we're going to add a large number of these options, and perhaps use a dict instead if that's the case, then maybe a vendor method to validate the dict against the available features or something?

File: etc/ansible/roles/network-runner/providers/nxos/conf_access_port.yaml [hunk 2]
```diff
@@ -50,6 +50,15 @@
         running_config: "{{ output.stdout[2] }}"
       connection: network_cli
 
+    - name: "nxos: enable portfast"
+      nxos_config:
+        lines:
+          - "spanning-tree portfast"
```
Comment (naved001) [inline etc/ansible/roles/network-runner/providers/nxos/conf_access_port.yaml]: FYI, I did not see this command on our nexus switch to enable portfast. This is how we do it:
```
r4-pA-c23-nexus3548(config-if)# spanning-tree port
port            port-priority   
r4-pA-c23-nexus3548(config-if)# spanning-tree port type 
edge      network   normal    
r4-pA-c23-nexus3548(config-if)# spanning-tree port type edge trunk 
Warning: Edge port type (portfast) should only be enabled on ports connected to a single
 host. Connecting hubs, concentrators, switches, bridges, etc... to this
 interface  when edge port type (portfast) is enabled, can cause temporary bridging loops.
 Use with CAUTION

r4-pA-c23-nexus3548(config-if)#
```

Specs of our nexus switch:
```
Software
  BIOS: version 5.1.0
 NXOS: version 9.2(3)
  BIOS compile time:  05/20/2018
  NXOS image file is: bootflash:///nxos.9.2.3.bin
  NXOS compile time:  2/17/2019 5:00:00 [02/17/2019 15:07:27]
Hardware
  cisco Nexus3500 C3548P-10G Chassis 
  Intel(R) Pentium(R) CPU  @ 1.50GHz with 4024204 kB of memory.
  Processor Board ID FOC18028M4C
```
cc: @tzumainn

**Comment by radez** [comment]
thanks for the actual command Naved :)
This was a mock up, we hadn't verified the command yet.

**Comment by naved001** [comment]
@radez oh cool, was just looking at the PRs and thought I would share what I know!

**Comment by radez** [comment]
Glad you did!

**Comment by michaeltchapman** [comment]
Needs some unit tests, otherwise looks good

**Pull request merged**